CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ include(rapids-cuda)
 include(rapids-export)
 include(rapids-find)
 
-set(legate_core_version 23.07.00)
+set(legate_core_version 23.09.00)
```

Comment:
This should come from a merge of 23.09?

Reply:
Or is it something I forgot?

Reply:
This change was made before the default branch was switched to 23.09. I have rebased my code now.